knowledge-base/importing-fiddler-classic-settings.md

```diff
@@ -0,0 +1,68 @@
+type: how-to
+slug: fe-migrating-from-classic
+publish: true
+res_type: kb
+---
+
+The foundation of the Fiddler Everywhere is built upon the old Fiddler Classic application. While the FIddler Classic is still available, more and more users are choosing to migrate to Fiddler Everywhere (as noted on the official product page, the Fiddler Classic application is no longer in active development and offers no commitments for releases, patches or tech support). There are multiple reasons why using Fiddler Everywhere should be your preferred choice, and the below feature comparison table demonstrates some of them:
+
+| Feature          | Fiddler Classic   | Fiddler Everywhere     | 
+|:-----------------|:------------------|:-----------------------|
+| OS support       | Windows only      | Windows, macOS, Linux  |
+| System proxy     | Windows only      | Windows, macOS, Linux  |
+| Low-level network capturing | No     | Yes (macOS) |
+| Sandboxed browser capturing | No     | Yes |
+| Rules to modify traffic | AutoResponder (limited options) | Rules (new enhanced functionality) |
+| Comparing HTTP sessions | Third-party extensions    | Built-in Comparison tab |
+| Filters | Limited pre-defined options | Advanced filtering options |
+| Collaboration | Manual export | Sharing, local & cloud saves, and manual export options |
+| Supported version of HTTP  | HTTP/1.x | HTTP/1.x and HTTP/2.x |
+| Maximum supported version of TLS  | TLS 1.2 | TLS 1.3 |
+| Support for streaming protocols  | WebSocket | WebSocket, SSE, Socket.IO, gRPC |
+| Statistical and analytical data | Statistics tab | Overview tab with enhanced statistical data and diagram |
+| Enterprise options | No | Technical support, SSO login, offline mode, managed app configuration, and more |
+
+The comparison table showcases several reasons why you would like to migrate from Fiddler Classic to the new Fiddler Everywhere. In this article, we will provide information about the built-in import wizard available in Fiddler Everywhere, which enables you to quickly and efficiently import your settings, rules, and filters from the old Fiddler Classic.
+
+## Importing Application Options
+
+Fiddler Everywhere version 5.2.0 provides a built-in wizard allowing you to import the following configuration options from Fiddler Classic automatically. All configuration options have full feature parity meaning they behave exactly the same in both applications. Once imported the configuration options can be found in the **Settings** menu in Fiddler Everywhere.
+
+The import wizard is accessible through the **Home** pane in Fiddler Everywhere.
+![import configuration options](../images/import-from-classic/import-options.png)
+
+- Fiddler Classic certificate&mdash;Instructs Fiddler Everwhere to use the installed Fiddler Classic certificate authority file. This option is available only if you have not yet installed and trusted the Fiddler Everywhere CA.
```

Comment:
The certificate checkbox will also be disabled if Fiddler Classic's certificate is not trusted ot is not compatible with Fiddler Everywhere. Fiddler Classic's certificate is compatible if it was created with BCCertMaker:
<img width="593" alt="image" src="https://github.com/user-attachments/assets/4f1e25c2-14a9-477a-a041-1716c5522a1f">